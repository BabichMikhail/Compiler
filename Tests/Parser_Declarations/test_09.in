var
	abc : integer = 10;