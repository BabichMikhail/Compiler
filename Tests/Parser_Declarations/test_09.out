Var	abc
Type	integer
10

